README.md

```diff
@@ -10,3 +10,7 @@ Data collection scripts are kept in `./collection-scripts`.  The content of that
 The data collection scripts should only include collection logic for components that are included as part of the OpenShift
 CVO payload.  Outside components are encouraged to produce a similar "must-gather" image, but this is not the spot to be
 included.
+
+## How to run
```

Comment:
Should we link to https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data (or something similar on how to run advanced commands, later  in these docs)?

Reply:
Sure, that may be useful